pkg/apiserver/apiserver.go

```diff
@@ -193,6 +195,9 @@ func NewConfig(
 }
 
 func (c *Config) Complete(informers informers.SharedInformerFactory) completedConfig {
+	c.genericConfig.OpenAPIV3Config = genericapiserver.DefaultOpenAPIV3Config(
+		openapi.GetOpenAPIDefinitions,
+		genericopenapi.NewDefinitionNamer(Scheme))
```

Comment:
This should be set in `createAPIServerConfig` where `OpenAPIConfig` is set.

Reply:
Thanks for the reminder, moved these lines to `cmd/antrea-controller/controller.go`